cmd/client/status.go

```diff
@@ -52,7 +51,7 @@ var StatusCmd = &cobra.Command{
 			exitOnError(fmt.Errorf("Failed to get status, %s: %s", resp.Status, data))
 		}
 
-		var status analyzer.Status
+		var status interface{}
```

Comment:
is there any issue with the current implementation ? why changing analyzer.Status to interface{} ?

Reply:
The StatusReporter interface is
```
type StatusReporter interface {
        GetStatus() interface{}
}
```
So the returned value has to be interface{} otherwise it does not fulfil the interface